packages/pretty-format/src/index.js

```diff
@@ -146,7 +146,7 @@ function printBasicValue(
     return printSymbol(val);
   }
   if (toStringed === '[object Date]') {
-    return toISOString.call(val);
+    return isNaN(+val) ? 'Date { NaN }' : toISOString.call(val);
```

Comment:
We could also keep around `toString` and fall back to that one. It prints `Invalid Date`.


```js
const toStringValue = toString.call(val);

return toStringValue === 'Invalid Date' ? toStringValue : toISOString.call(val);
```

Reply:
too late :-p
anyway, I do not like testing over a "human" string. While an invalid date will always cast to the wonderful JavaScript number... not a number 🤣

Reply:
Hah, too late 😀 

Feel free to send a new PR with this change if you think it makes sense